functions/java-jre.bash

```diff
@@ -212,6 +251,40 @@ java_zulu_tar_update_available(){
   return 0
 }
 
+## Install Azul's Java Zulu Enterprise using APT repository.
+## (package manager distributions only available on x86-64bit platforms when checked in April 2020)
+## Valid arguments: "8" or "11"
+##
+##    java_zulu_enterprise_apt(String ver)
+##
+java_zulu_enterprise_apt(){
+  if [ "$1" == "8" ]; then
+    if ! dpkg -s 'zulu-8' 2>&1> /dev/null; then # Check if already is installed
```

Comment:
is that the official Azul package name ?

Reply:
yes